Updated Get Started docs (#1716)
opensource-observer/oso: apps/docs/docs/get-started/index.mdx

Feel free to name this GCP project anything you'd like.
(Or you can simply leave the default project name 'My First Project'.)

> **Also note that you won't be able to create a new project if you're not an administrator to your e-mail address**

Reviewer: ```suggestion
*Note: you won't be able to create a new project if you're not an administrator of your Google organization*
```

Reviewer: FYI I made a minor wording change here @karibaEA
